### PR TITLE
Updated configurability

### PR DESCRIPTION
**Pull request by ronaldsteen**
Hi, 

I've updated configurability. You can now specify pins and addresses.

Cheers,
Ronald

### CODE REVIEW

File: src/board.coffee [hunk 1]
```diff
@@ -146,17 +146,38 @@ class Board extends events.EventEmitter
...
+          temperature: parseFloat(args) 
+        })
+    return promise
+  
+  #DST [pin] all -> all temperatures on that pin.
+  readDstAll: (pin) ->
+    reading = @writeAndWait("DST #{pin} all\n")
+    promise = reading
+      .then( (args) -> 
+        console.log "readDstAll args: ", args
```
Comment (sweetpi) [inline src/board.coffee]: Could you please remove all `console.log`s?

File: package.json [hunk 2]
```diff
@@ -19,12 +19,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/pimatic/homeduinojs.git"
+    "url": "git://github.com/ronaldsteen/homeduinojs-dst.git"
   },
-  "author": "Oliver Schneider <oliverschneider89+sweetpi@gmail.com>",
```
Comment (sweetpi) [inline package.json:24]: > The "author" is one person. "contributors" is an array of people. 

https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Could you please add your name to `contributors` instead?
Please leave the issue and main url as it wars.

**Comment by ronaldsteen** [comment]
author, contributors and console.log entries are fixed.

**Comment by sweetpi** [comment]
Big thanks.

**Pull request merged**